xivo_dao/resources/line/search.py

```diff
@@ -15,7 +19,37 @@
                                'device_slot': LineFeatures.num,
                                'protocol': LineFeatures.protocol,
                                'device_id': LineFeatures.device,
-                               'name': LineFeatures.name},
+                               'name': LineFeatures.name,
+                               'caller_id_name': LineFeatures.caller_id_name,
+                               'caller_id_num': LineFeatures.caller_id_num,
+                               'exten': Extension.exten},
                       default_sort='name')
 
-line_search = SearchSystem(config)
+
+class LineSearchSystem(SearchSystem):
+
+    def search_from_query(self, query, parameters):
+        if 'exten' in parameters and isinstance(parameters['exten'], (str, six.text_type)):
+            extens = parameters.pop('exten').split(',')
```

Comment:
There is a reason to search by mutli exten? Ticket doesn't mention that
search example by single exten: https://github.com/wazo-platform/xivo-dao/commit/88a371e12691d810b402b8729b7d60b36b56b6ae